Nicholas Mather/week01/word_guesser/js/exercises.js

```diff
@@ -0,0 +1,77 @@
+// # Homework: The Word Guesser
+//
+// You'll create a simple word guessing game where the user gets infinite tries to guess the word (like Hangman without the hangman, or like Wheel of Fortune without the wheel and fortune).
+//
+// - Create two global arrays: one to hold the letters of the word (e.g. 'F', 'O', 'X'), and one to hold the current guessed letters (e.g. it would start with '\_', '\_', '\_' and end with 'F', 'O', 'X').
+
+const targetWord = ["r", "e", "d"];
+const guessedLetters = ['_','_','_'];
```

Comment:
might have been better to initialise this as just an empty array, rather than storing underscores. 
i.e. `const guessedLetters = [];`